dali/test/python/test_RN50_data_pipeline.py

```diff
@@ -210,6 +211,18 @@ def define_graph(self):
                     help='Shuffle data when reading')
 parser.add_argument('--simulate_N_gpus', default=None, type=int, metavar='N',
                     help='Used to simulate small shard as it would be in a multi gpu setup with this number of gpus. If provided, each gpu will see a shard size as if we were in a multi gpu setup with this number of gpus')
+parser.add_argument('--remove_default_pipeline_paths', action='store_true',
+                    help="For all data pipeline types, remove the default values")
+parser.add_argument('--file_read_pipeline_paths', default=None, type=str, metavar='N',
+                    help='Override default FileReadPipeline paths. Separate multiple paths by commas')
```

Comment:
Nitpick: it doesn't override the paths but adds additional ones.

Reply:
I'm fine with either clearing the dictionary entry or rewording this.

Reply:
Updated to "Add custom". Hope that covers it well enough!